0.download_data/scripts/nbconverted/download_data.py

```diff
@@ -9,66 +9,39 @@
 import pandas as pd
 import pathlib
```

Comment:
```suggestion
import pathlib
import pandas as pd
```
According to [PEP8](https://peps.python.org/pep-0008/#imports) there is a specific order how imports should be formatted. 

From PEP8 Guide: 
Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants. Imports should be grouped in the following order:
-  Standard library imports.
 - Related third party imports.
 - Local application/library specific imports.
You should put a blank line between each group of imports.

Reply:
Thanks for sharing this, I will keep it in mind for my future code!